src/components/flowchart/styles/_layers.scss

```diff
@@ -69,8 +69,16 @@
   display: flex;
   align-items: center;
   height: 20px;
-  padding-left: 15px;
+  padding-left: 18px;
   font-weight: bold;
   font-size: 1.6em;
   white-space: nowrap;
+
+  .kui-theme--dark & {
+    color: #b5bdc2;
+  }
+
+  .kui-theme--light & {
+    color: #4e5861;
+  }
```

Comment:
I've gone through and refactored colours to use opacity instead then, as well as moving some in to variables.

Reply:
Thanks! I've added a comment about this on `src/components/app/app.scss`, but basically, as much as possible I'd recommend using the `opacity` css rule rather than `fill`/`color`/`stroke` etc, so that these icons can inherit colours via the cascade and just modify their root colour, and reduce the number of places where it might need to be changed, and the amount of overrides.